guides/doc-Content_Management_Guide/master.adoc

```diff
@@ -8,6 +8,8 @@ endif::[]
 include::common/attributes.adoc[]
 :context: content-management
 
+include::topics/QuickStart.adoc[]
```

Comment:
> This guide assumes provisioning occurs but I can't find that. Perhaps link to the provisioning guide?

Yes. Provisioning via foreman is kind of a different thing. This is more about "What would I need to do to make my foretello instance  content enabled/ready so that any host (not just the ones I provisioned via foreman) can consume content off it"

Reply:
> Is your new chapter targeted at the community/upstream moreso than Satellite users?

Intended audience is an upstream user who has already installed katello and wants to get started on pulling content from upstream to their instance and use the synced content to manage a host in the network.

Idea is to just get them introduced to the basic entities. Now I could add a "For more information on what products/repositories/subscriptions/cv/ak go to the chapter referring those things." If that would help.